examples/problem_decomposition/ifci_mifno.ipynb

```diff
@@ -0,0 +1,554 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Exploring the Method of Increments with QEMIST Cloud and Tangelo\n",
+    "\n",
+    "In this notebook, we illustrate how users can leverage QEMIST Cloud and Tangelo to explore the impact of quantum computing on problems tackled with the method of increments (MI) problem decomposition technique, which includes the Frozen Natural Orbital Based Method of Increments (MI-FNO) and incremental Full Configuration Interaction (iFCI). \n",
+    "\n",
+    "You do not need to have the `qemist-client` python package installed to run this notebook: only `tangelo` is required. For more information about [QEMIST Cloud](https://goodchemistry.com/qemist-cloud/) (installation, features, issues, `qemist_client` API...), please refer to the QEMIST Cloud documentation or contact the development team.\n",
+    "\n",
+    "The first section provides a high-level description of the MI approach. The second one briefly shows how QEMIST Cloud can apply this approach to a usecase, and provide reference results computed with high-accuracy classical solvers. We then focus on the API provided in Tangelo allowing users to combine the MI performed by QEMIST Cloud and any quantum workflow written with Tangelo.\n",
+    "\n",
+    "This notebook assumes that you already have installed Tangelo in your Python environment, or have updated your Python path so that the imports can be resolved. If not, executing the cell below installs the minimal requirements for this notebook."
```

Comment:
I'd replace all that by just

The cell below installs the required dependencies for this notebook in case Tangelo is not found in your current environment.